Services/InputRepositories/CoSignatureQualityRepository.cs

```diff
@@ -34,22 +34,21 @@ public CoSignatureQualityRepository(Uri uri, NetworkCredential networkCredential
         }
         public async Task<IEnumerable<Domain.Entities.CoSignatureQuality>> GetCoSignatureQualitiesAsync(int omItemNumber, int versionNumber, int coSingnatureId)
         {
-            using (HttpClient client = new HttpClient(new HttpClientHandler { UseDefaultCredentials = true }))
+            using (var client = new HttpClient(new HttpClientHandler { UseDefaultCredentials = true }))
```

Comment:
HttpClient should be singleton. Create in the di.

Reply:
Move to separate PR. See #59